Equipo2Hosteleria/src/main/java/Modelo/ObtenerDatos.java

```diff
@@ -0,0 +1,44 @@
+package Modelo;
+
+public class ObtenerDatos {
+	static Producto arrayProducto[]=new Producto[5];
+	static String arrayNombresProducto[]=new String[5];
+	static double arrayVentaProducto[]=new double[5];
+	
+	public static Producto[] listaProductos() {
+		arrayProducto[0] = new Producto("Garbanzos","Comida","30-01-2021",5,5);
+		arrayProducto[1] = new Producto("Paella","Comida","31-01-2021",5,5);
+		arrayProducto[2] = new Producto("Bacalao","Comida","02-02-2021",5,5);
+		arrayProducto[3] = new Producto("Hamburguesa","Comida","30-01-2021",5,5);
+		arrayProducto[4] = new Producto("Cerveza","Bebida","01-02-2021",5,5);
+		return arrayProducto;
+	}
+	
+	public static String[] nombresProductos() {
```

Comment:
Porque usáis siempre métodos estáticos?

Reply:
Una confusión que ya ha sido solucionada.